### PR TITLE
feat: ✨checkbox label slot support use option label

### PR DESCRIPTION
**Pull request by waldonUB**
See this issue for details: https://github.com/vueComponent/ant-design-vue/issues/6641

### CODE REVIEW

**Comment by waldonUB** [comment]
The demo test run failed, should I update the demo.test.js.snap file and commit?

**Comment by waldonUB** [comment]
✓ renders ./components/checkbox/demo/group.vue correctly (89 ms)

File: components/checkbox/Group.tsx [hunk 1]
```diff
@@ -100,7 +100,7 @@ export default defineComponent({
             onChange={option.onChange}
             class={`${groupPrefixCls}-item`}
           >
-            {option.label === undefined ? slots.label?.(option) : option.label}
```
Comment (tangjinzhou) [inline components/checkbox/Group.tsx:103]: 算是破环性更新了，合并到 v4 分支了，v3 这个不动了

**Review by tangjinzhou — APPROVED** [review]

File: components/checkbox/Group.tsx (hunk 1 above)
Comment (waldonUB) [reply components/checkbox/Group.tsx:111]: OK

**Pull request merged**